src/overlays/actors/ovl_Object_Kankyo/z_object_kankyo.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_object_kankyo.c
  * Overlay: ovl_Object_Kankyo
- * Description: Snow, rain in Skull Kid backstory cutscene, and bubbles deep in Pinnacle Rock
+ * Description:
```

Comment:
Any particular reason the description was deleted? I'm guessing this was just a git mistake?